docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -7,7 +7,7 @@ ms.date: 10/31/2023
 
 # Creating metrics
 
-**This article applies to: ✔️** .NET Core 6 and later versions **✔️** .NET Framework 4.6.1 and later versions
+**This article applies to: ✔️** .NET Core 6 and later versions **✔️** .NET Framework 2 and later versions
```

Comment:
Do we support metrics in Framework 2.0??

Reply:
Good catch - this should read "4.6.2" not "2".  I'll fix this